src/etsn20_lab4/str_search.java

```diff
@@ -6,9 +6,9 @@
 	public static void main(String[] args) {
 		
 	    Scanner userInput = new Scanner(System.in);
```

Comment:
No need to user Scannner. You can use Strings[] args as user input. Or atleast give them the options to use them